src/briefcase/platforms/android/gradle.py

```diff
@@ -282,6 +282,24 @@ def permissions_context(self, app: AppConfig, x_permissions: dict[str, str]):
             "features": features,
         }
 
+    def _write_requirements_file(
+        self, app: AppConfig, requires: list[str], requirements_path: Path
+    ):
+        super()._write_requirements_file(app, requires, requirements_path)
+
+        # Flatpak runs ``pip install`` using an ``install_requirements.sh`` which Briefcase uses
+        # to indicate user-configured arguments to the command
+        pip_options = "\n".join(
+            [f"# Generated {datetime.datetime.now()}"] + self._extra_pip_args(app)
+        )
```

Comment:
Unlike the requirements file, the options file doesn't need a timestamp comment to trigger a re-run of pip when it changes, because it's read by Gradle on every build. And if we use this file format in Flatpak as I suggested [above](https://github.com/beeware/briefcase/pull/2059#discussion_r1844710348), then eliminating the comment would make it easier to process with a shell script.

Reply:
It's not needed to trigger a re-run, but I'd argue it's helpful as a diagnostic tool. However, if it interferes with easy command line processing, then it's a "nice-to-have" we can definitely lose.